deeposlandia/webapp.py

```diff
@@ -0,0 +1,153 @@
+"""Flask web application for deeposlandia
+"""
+
+import daiquiri
+from flask import (abort, Flask, jsonify, redirect,
+                   render_template, request, send_from_directory, url_for)
+import json
+import logging
+import numpy as np
+import os
+from PIL import Image
+from werkzeug.utils import secure_filename
+
+from deeposlandia import utils
+from deeposlandia.inference import predict
+
+MODELS = ('feature_detection', 'semantic_segmentation')
+DATASETS = ('mapillary', 'shapes')
+ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg'])
+PROJECT_FOLDER = '/tmp/deeposlandia'
+UPLOAD_FOLDER = os.path.join(PROJECT_FOLDER, 'uploads/')
+PREDICT_FOLDER = os.path.join(PROJECT_FOLDER, 'predicted/')
+os.makedirs(UPLOAD_FOLDER, exist_ok=True)
+os.makedirs(PREDICT_FOLDER, exist_ok=True)
+
+daiquiri.setup(level=logging.INFO)
+logger = daiquiri.getLogger("deeposlandia-webapp")
+
+app = Flask(__name__)
+app.config['ERROR_404_HELP'] = False
+app.config['SWAGGER_UI_DOC_EXPANSION'] = 'list'
```

Comment:
I think you don't use Swagger, you can remove it.

Reply:
Done.